pom.xml

```diff
@@ -504,7 +504,7 @@
         <version.org.testcontainers>1.16.0</version.org.testcontainers>
         <version.org.testng>6.14.3</version.org.testng>
         <version.org.wildfly.arquillian>3.0.1.Final</version.org.wildfly.arquillian>
-        <version.org.wildfly.core>19.0.0.Beta2</version.org.wildfly.core>
+        <version.org.wildfly.core>19.0.0.Beta3-SNAPSHOT</version.org.wildfly.core>
```

Comment:
FYI this looks like it crept in by mistake.

Reply:
Yes, it was a mistake. I just fixed that @darranl .